### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by jetzlstorfer**

### CODE REVIEW

File: content/docs/develop/installation/setup-keptn/index.md [hunk 1]
```diff
@@ -49,11 +45,11 @@ Select one of the following options:
     CLUSTER_NAME=nameofcluster
     ZONE=us-central1-a
     REGION=us-central1
-    GKE_VERSION="1.12.7-gke.10"
+    GKE_VERSION="1.18.8-gke.10"
```
Comment (dirkwall) [inline content/docs/develop/installation/setup-keptn/index.md]: There is no version 1.18.8

File: content/docs/develop/installation/upgrade-keptn/index.md [hunk 2]
```diff
@@ -6,84 +6,22 @@ icon: setup
...
 
-## Prerequisites
-   
-- Please note that all [command line tools](../setup-keptn-gke#prerequisites) are needed when upgrading keptn.
-  
-    - Additionally, [yq](https://github.com/mikefarah/yq) is required.
+## Uninstall keptn
 
-- Furthermore, please note that we increaded the GKE cluster size to one `n1-standard-16` node.
+For uninstalling keptn, [download this script](https://github.com/keptn/installer/blob/0.4.0/scripts/common/uninstallKeptn030.sh) and execute it:
```
Comment (dirkwall) [inline content/docs/develop/installation/upgrade-keptn/index.md]: Are we uninstalling Dynatrace or any other components that not belong to keptn core during that process?

File: content/docs/develop/installation/setup-keptn/index.md (hunk 1 above)
Comment (jetzlstorfer) [reply content/docs/develop/installation/setup-keptn/index.md]: Seems like a typo. Will change it to 1.12.8

File: content/docs/develop/installation/upgrade-keptn/index.md (hunk 2 above)
Comment (jetzlstorfer) [reply content/docs/develop/installation/upgrade-keptn/index.md]: No it does only uninstall keptn core.

**Pull request merged**